governance.md

```diff
@@ -163,7 +160,7 @@ community meeting.
 
 All repositories under Kubernetes github orgs, such as kubernetes and kubernetes-incubator,
 should follow the procedures outlined in the [incubator document](incubator.md). All code projects
```

Comment:
Also, this line links to https://github.com/kubernetes/community/blob/master/incubator.md but that document is deprecated.

Changing this needs to go through proper channels though and is out of scope for this PR.